src/core/client/admin/routes/Community/UserTable.tsx

```diff
@@ -28,56 +29,86 @@ interface Props {
   loading: boolean;
 }
 
-const UserTable: FunctionComponent<Props> = props => (
-  <>
-    <HorizontalGutter size="double">
-      <Table fullWidth>
-        <TableHead>
-          <TableRow>
-            <Localized id="community-column-username">
-              <TableCell className={styles.usernameColumn}>Username</TableCell>
-            </Localized>
-            <Localized id="community-column-email">
-              <TableCell className={styles.emailColumn}>
-                Email Address
-              </TableCell>
-            </Localized>
-            <Localized id="community-column-memberSince">
-              <TableCell className={styles.memberSinceColumn}>
-                Member Since
-              </TableCell>
-            </Localized>
-            <Localized id="community-column-role">
-              <TableCell className={styles.roleColumn}>Role</TableCell>
-            </Localized>
-            <Localized id="community-column-status">
-              <TableCell className={styles.statusColumn}>Status</TableCell>
-            </Localized>
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {!props.loading &&
-            props.users.map(u => (
-              <UserRowContainer key={u.id} user={u} viewer={props.viewer!} />
-            ))}
-        </TableBody>
-      </Table>
-      {!props.loading && props.users.length === 0 && <EmptyMessage />}
-      {props.loading && (
-        <Flex justifyContent="center">
-          <Spinner />
-        </Flex>
-      )}
-      {props.hasMore && (
-        <Flex justifyContent="center">
-          <AutoLoadMore
-            disableLoadMore={props.disableLoadMore}
-            onLoadMore={props.onLoadMore}
-          />
-        </Flex>
-      )}
-    </HorizontalGutter>
-  </>
-);
+const UserTable: FunctionComponent<Props> = props => {
+  const [userDrawerID, setUserDrawerId] = useState("");
```

Comment:
Would read better to be `userDrawerUserID`, even with the repetition, otherwise it reads as "the id of the drawer", which is confusing.

Reply:
Good idea, I've updated this. Also switched setUserDrawerId -> setUserDrawerID as I know you prefer ID to Id.